src/operations/indexes.ts

```diff
@@ -168,12 +167,9 @@ export class CreateIndexesOperation extends CommandOperation<Document> {
     this.collectionName = collectionName;
 
     this.indexes = indexes;
-    if (indexes.length === 1) {
-      this.onlyReturnNameOfCreatedIndex = true;
-    }
   }
 
-  execute(server: Server, session: ClientSession, callback: Callback<Document>): void {
+  execute(server: Server, session: ClientSession, callback: Callback<any>): void {
```

Comment:
This is cheating :) But I think you can just use `Callback<T>` here since you default to `string[]`

Reply:
I spent a while on this and couldn't figure out how to get around using `any`, with `T` there it gives this error:

```
Argument of type '(string | undefined)[]' is not assignable to parameter of type 'T'.
  '(string | undefined)[]' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint '{}'.ts(2345)
```

Reply:
I spent a while on this and couldn't figure out how to get around using `any`, with `T` there it gives this error:

```
Argument of type 'string[]' is not assignable to parameter of type 'T'.
  'string[]' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint '{}'.ts(2345)
```

Reply:
fixed!